docs/components/api/page-wrapper.js

```diff
@@ -21,8 +20,7 @@ export default class Api extends React.Component {
 
 Api.propTypes = {
     frontMatter: PropTypes.shape({
-        title: PropTypes.string.isRequired,
-        hideFeedback: PropTypes.bool
```

Comment:
This frontMatter is no longer needed now that the feedback component has been removed completely.